example-article.json

```diff
@@ -11,11 +11,29 @@ This file specifies what posts look like:
 # http://service/author/{AUTHOR_ID}/posts (all posts made by {AUTHOR_ID} visible to the currently authenticated user)
 # or
 # http://service/posts/{POST_ID} access to a single post with id = {POST_ID}
+# or
+# http://service/posts/{post_id}/comments access to the comments in a post
 #
 # All of the previous URIs will get a list of posts like this:
 
+# If a page is provided, return the results for that page, if no page is provided return page 0
+# If a size is provided, return pages in this size up to the maximum set on the server. If no size is provided
+# return the default page size
+# Use same pagination style for all endpoints in this file
+# Example:
+# GET http://service/author/posts?page=4&size=50
```

Comment:
@derekdowling I don't see how offering offset vs page is any different - Your offset would be page # * count, in a truly dynamic web site where the count is changing continuously I would use something like last_guid = {last_guid_Previous_page} as to prevent the pages from changing as a user moves through them.